website/docs/reference/warehouse-setups/vertica-setup.md

```diff
@@ -28,16 +35,27 @@ Some core functionality may be limited. If you're interested in contributing, ch
     <li><strong>dbt Cloud support</strong>: {frontMatter.meta.cloud_support}</li>
     <li><strong>Minimum data platform version</strong>: {frontMatter.meta.min_supported_version}</li>
     </ul>
+
+
 <h2> Installing {frontMatter.meta.pypi_package} </h2>
-pip is the easiest way to install the adapter:
-<code>pip install {frontMatter.meta.pypi_package}</code>
-<p>Installing <code>{frontMatter.meta.pypi_package}</code> will also install <code>dbt-core</code> , <code>dbt-tests-adapter</code> and any other dependencies.</p>
+
+pip is the easiest way to install the adapter: <code>pip install {frontMatter.meta.pypi_package}</code>
+
+<p>Installing <code>{frontMatter.meta.pypi_package}</code> will also install <code>dbt-core</code> and any other dependencies.</p>
+
 <h2> Configuring {frontMatter.meta.pypi_package} </h2>
-<p>For dbt-vertica-specifc configuration please refer to <a href={frontMatter.meta.config_page}>{frontMatter.meta.platform_name} Configuration</a> </p>
+
+<p>For {frontMatter.meta.pypi_package} specific configuration please refer to <a href={frontMatter.meta.config_page}>{frontMatter.meta.platform_name} Configuration</a> </p>
+
 <p>For further info, refer to the GitHub repository: <a href={`https://github.com/${frontMatter.meta.github_repo}`}>{frontMatter.meta.github_repo}</a></p>
-### Connecting to Vertica with **dbt-vertica**
+
+
+### Connecting to Vertica with {frontMatter.meta.pypi_package} 
```

Comment:
for inane reasons, you can't use markdown headers when injecting front matter info
```suggestion
<h3> Connecting to {frontMatter.meta.platform_name}   with {frontMatter.meta.pypi_package} </h3>
```